lib/schema.js

```diff
@@ -2251,6 +2251,61 @@ function _deletePath(schema, name) {
   delete branch[last];
 }
 
+/**
+ *
+ * @param {String|Array} path The virutal path(s) to remove.
+ * @returns {Schema} the Schema instance, or a mongoose error if the virtual does not exist.
+ * @api public
+ */
+
+Schema.prototype.removeVirtual = function(path) {
+  if (typeof path === 'string') {
+    path = [path];
+  }
+  if (Array.isArray(path)) {
+    path.forEach(function(name) {
+      if (this.virtuals[name] == null && !this.nested[name]) {
+        throw new MongooseError('Attempting to remove virtual path that does not exist.');
+      }
+      if (this.nested[name]) {
```

Comment:
This is my mistake. I thought that if a virtual for whatever reason was nested it would go there. Evidently, it seems that you can't nest a virtual as when I attempted to write a test to do so it was not registered in the schema anywhere.